main/libressl/APKBUILD

```diff
@@ -29,19 +29,21 @@ build() {
 		--prefix=/usr \
 		--sysconfdir=/etc \
 		--mandir=/usr/share/man \
-		--localstatedir=/var \
-		|| return 1
-	make || return 1
+		--localstatedir=/var
+	make
 }
 
 package() {
 	cd "$builddir"
-	make DESTDIR="$pkgdir" install || return 1
+	make DESTDIR="$pkgdir" install
 	install -Dm644 COPYING "$pkgdir"/usr/share/licenses/$pkgname/COPYING
+	# fix curl: (77) error setting certificate verify locations:
+	mkdir -p "$pkgdir"/etc/ssl/certs
+	ln -s /etc/ssl/cert.pem "$pkgdir"/etc/ssl/certs/ca-certificates.crt
```

Comment:
I think a relative symlink would be better, like `ln -s ../cert.pem "$pkgdir"/etc/ssl/certs/ca-certificates.crt`.